clusterbuster

```diff
@@ -2415,6 +2435,63 @@ EOF
     done
 }
 
+function create_vm_deployment() {
+    local namespace=$1
+    local instance=$2
+    local -i replicas=$4
+    local -i replica=0
+    local depname="${namespace}-${workload}-${instance}"
+    while (( replica++ < replicas )) ; do
+        local name="${depname}-${replica}"
+        local vmname=$(mkpodname "$name")
+        create_object -n "$namespace" "$name" <<EOF
+apiVersion: kubevirt.io/v1alpha3
+kind: VirtualMachine
+metadata:
+$(indent 2 standard_labels_yaml)
+  name: "$vmname"
```

Comment:
Use mkpodname here so that it's possible to prefix names with something else if the user wants.  You might want to add a `--vmprefix` option as well that does the same thing (use the `pod_prefix variable, even if the option is --vmprefix).

Nit: put the name first (before the labels).  It doesn't strictly matter, but having the name first is easier to read.